server/athenian/api/openapi/openapi.yaml

```diff
@@ -6830,3 +6882,92 @@ components:
       - developer_id
       - jira_name
       type: object
+    NotifyReleaseRequest:
+      additionalProperties: false
+      description: Request body of `/notify/release`.
+      example:
+      - author: Vadim Markovtsev
+        commit: c3f4d3a
+        name: 0.8.66
+        repository: github.com/athenianco/athenian-api
+        published_at: 2021-02-23T15:17:07.000+01:00
+        url: https://github.com/athenianco/athenian-api/releases/tag/v0.8.66
+      properties:
+        account:
+          description: Account ID.
+          type: integer
+        notifications:
+          items:
+            $ref: '#/components/schemas/ReleaseNotification'
+          type: array
+      required:
+      - account
+      - notifications
+      type: object
+    CommitHash:
+      description: Commit hash, either short (7 chars) or long (40 chars) form.
+      maxLength: 40
+      minLength: 7
+      type: string
+    ReleaseNotification:
+      additionalProperties: false
+      example:
+        author: Vadim Markovtsev
+        commit: c3f4d3a
+        name: 0.8.66
+        repository: github.com/athenianco/athenian-api
+        published_at: 2021-02-23T15:17:07.000+01:00
+        url: https://github.com/athenianco/athenian-api/releases/tag/v0.8.66
+      description: Push message about a custom release event.
+      properties:
+        repository:
+          $ref: '#/components/schemas/RepositoryID'
+        commit:
+          $ref: '#/components/schemas/CommitHash'
+        name:
+          description: Release name.
+          type: string
+        author:
+          description: Release author.
+          type: string
+        url:
+          description: Release URL.
+          format: url
+          type: string
+        published_at:
+          description: When the release was created. If missing, set to `now()`.
+          format: date-time
+          type: string
+      required:
+      - commit
+      - repository
+      type: object
+    NotifiedReleaseStatuses:
+      description: Status of the accepted release notifications.
+      items:
+        $ref: '#/components/schemas/NotifiedReleaseStatus'
+      type: array
+    NotifiedReleaseStatus:
+      additionalProperties: false
+      description: Status of the accepted release notification.
+      properties:
+        repository:
+          $ref: '#/components/schemas/RepositoryID'
+        commit:
+          $ref: '#/components/schemas/CommitHash'
+        status:
+          description: |-
+            Status of the release notification.
+            * `applied` the release commit was found and the release was successfully registered.
```

Comment:
I think, we shouldn't explicitly tell if we mapped something or not. We should think of this endpoint as async (as if it always returns `pending`).

Reply:
Fair enough, this is our implementation detail.

Reply:
Well, let's return nothing then. An empty response signals a successful operation, like in some other places.

Reply:
As mentioned in the other comment, we could return account ID. But I don't think it's useful, so yeah, we can return nothing, just 200.